debian/copyright

```diff
@@ -1,7 +1,29 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: mesaflash
+Upstream-Contact: LinuxCNC Developers <emc-developers@lists.sourceforge.net>
 Source: https://github.com/micges/mesaflash
 
+Files: *
 Copyright: 2013-2014 Michael Geszkiewicz
-License: GPL-2+
+License: GPL-2.0+
 
-The complete text of GPL version 2 can be found in the file
-/usr/share/common-licenses/GPL-2 on Debian systems.
+Files: debian/*
+Copyright: #YEAR# #USERNAME# <#EMAIL#>
+License: GPL-2.0+
+
+License: GPL-2.0+
```

Comment:
Is this duplication right/intended?

Reply:
Yes. 12 is a reference to a license and 14ff define it. This way you can have two File-Blocks reference the same license.